server/index.js

```diff
@@ -15,7 +15,16 @@ function SocketPeerServer(opts) {
   if (!opts.httpServer) {
     var host = opts.host || process.env.SOCKETPEER_HOST || process.env.HOST || '0.0.0.0';
     var port = opts.port || process.env.SOCKETPEER_PORT || process.env.PORT || 3000;
-    opts.httpServer = http.createServer();
+    opts.httpServer = http.createServer(function (req, res) {
+      var url = urllib.parse(req.url).pathname;
+
+      // For demo purposes.
+      if (url === '/' || url === '/demo/') {
```

Comment:
should probably have some option to disable this (or serve only for dev purposes)